packages/devtools_app/lib/src/screens/vm_developer/vm_service_private_extensions.dart

```diff
@@ -176,10 +176,67 @@ class Disassembly {
 /// An extension on [Func] which allows for access to VM internal fields.
 extension FunctionPrivateViewExtension on Func {
   static const _unoptimizedCodeKey = '_unoptimizedCode';
+  static const kindKey = '_kind';
```

Comment:
If these keys aren't used outside of these extensions, we should make them private. I haven't been terribly consistent about that, but I've been meaning to go back and make all these key variables private for all the extensions.

Reply:
I've made them private